README.md

```diff
@@ -1,92 +1,150 @@
-* [7-Zip](https://www.7-zip.org/download.html)
-* [Win Flex/Bison](https://sourceforge.net/projects/winflexbison/)
-* [NSIS](https://nsis.sourceforge.io/)
-* [NSIS LockedList Plugin](https://nsis.sourceforge.io/LockedList_plug-in)
-* [NSIS Inetc Plugin](https://nsis.sourceforge.io/Inetc_plug-in)
-* [NSIS Registry Plugin](https://nsis.sourceforge.io/Registry_plug-in)
-
-
-### Preparing system for building
-
- - Remove "C:\Strawberry\c\bin" from PATH.
- - Make sure no pkg-config utility is in PATH, this will cause conflicts with Strawberry's own pkg-config.
-   (del c:\strawberry\perl\bin\pkg-config.bat).
- - Make sure no MinGW-W64 (gcc, g++, etc) installation is in PATH, as this can cause those to be picked up as compiler instead of MSVC.
- - You need the sed and patch utility, add C:\Program Files\Git\usr\bin to PATH.
-
-
-### Installing NSIS plugins
-
-To create the Strawberry installer, you need 3 NSIS plugins (LockedList, Inetc and Registry).
-
-Those need to be manually extracted, and copied to C:\Program Files (x86)\NSIS\Plugins
-
-Specifically, the following files need to exist:
-
-    C:\Program Files (x86)\NSIS\Plugins\LockedList64.dll
-    C:\Program Files (x86)\NSIS\Plugins\Registry.dll
-    C:\Program Files (x86)\NSIS\Plugins\x86-unicode\LockedList.dll
-    C:\Program Files (x86)\NSIS\Plugins\x86-unicode\Registry.dll
-    C:\Program Files (x86)\NSIS\Plugins\x86-unicode\INetC.dll
-
-
-### Building all dependencies from source
-
-This guide provides a "download.bat" and "build.bat" file to automatically build all dependencies.
-
-Start "x64 Native Tools Command Prompt for VS 2022"
-
-Run download.bat
-
-This should download all necessary sources.
-
-
-#### Build for debug
-
-    build.bat debug
-
-
-#### Build for release
-
-    build.bat release
-
-
-### Building strawberry
-
-Add "c:\strawberry_msvc_x86_64_debug\bin" for the debug version, or "c:\strawberry_msvc_x86_64_release\bin" for the release version to PATH.
-
-Open the strawberry git source directory in Visual Studio 2022.
-
-Add the following extra CMake arguments:
-
-For debug:
-
-    -DARCH=x86_64 -DICU_ROOT=c:\\strawberry_msvc_x86_64_debug
-
-
-For release:
-
-    -DARCH=x86_64 -DICU_ROOT=c:\\strawberry_msvc_x86_64_release
+## Links
+
+- [Strawberry Music Player Website](https://www.strawberrymusicplayer.org)
+- [Wiki](https://wiki.strawberrymusicplayer.org/wiki/Main_Page)
+- [Forum](https://forum.strawberrymusicplayer.org)
+- [Github sources](https://github.com/strawberrymusicplayer/strawberry)
+- [Github MSVC](https://github.com/strawberrymusicplayer/strawberry-msvc)
+- [Github MSVC prebuilt binary dependencies](https://github.com/strawberrymusicplayer/strawberry-msvc-dependencies)
+
+## Manual installation on Windows
+
+Download and install manually:
+- [Git](https://git-scm.com/downloads)
+    - Default settings can be used
+- [Visual Studio 2022 Community](https://visualstudio.microsoft.com/vs/)
+    - Select `Desktop development with C++`
```

Comment:
Is this enough, does it install C++ CMake tools for Windows?

What about Qt VS Tools for Visual Studio 2022?

Reply:
> Is this enough, does it install C++ CMake tools for Windows?

Yes, when checking checkbox `Desktop devleopment with C++`, it installed all features/dependencies listed in the screenshot below, including `C++ CMake tools for Windows`. No other checkboxes are needed:

![image](https://github.com/user-attachments/assets/1a6afa87-0ef9-4f0d-a06f-4794fa6fae77)

> What about Qt VS Tools for Visual Studio 2022?

No, this is not installed automatically and not needed to build Strawberry debug or release. I think it is suseful for development.

Installation is described here: https://www.qt.io/blog/qt-vs-tools-for-visual-studio-2022.

Microsoft likes it to change interfaces, so in my version it looks like this:

![image](https://github.com/user-attachments/assets/9a3696fe-d5ce-41c7-b82a-34b91534b83a)